uint64 collision1
uint64 collision2

geometry_msgs/Vector3[] position
geometry_msgs/Vector3[] normal
float64[] depth
geometry_msgs/Wrench[] body1_wrench
geometry_msgs/Wrench[] body2_wrench

uint64 world